pkg/services/ngalert/api/tooling/Makefile

```diff
@@ -61,8 +61,8 @@ fix:
 	goimports -w -v $(GENERATED_GO_MATCHERS)
 
 clean: clean-go
-	rm spec.json
-	rm spec-stable.json
+	rm -f spec.json
+	rm -f spec-stable.json
```

Comment:
Drive-by improvement to the PR i made earlier. Now it won't error if you have already locally deleted one of these files.